script/bin/operator.rs

```diff
@@ -182,7 +185,7 @@ impl SP1AvailLightClientOperator {
                 .timeout(Duration::from_secs(900))
                 .run()?;
             info!("Generate proof end");
-            log::info!("Attempting to update to new head block: {:?}", latest_block);
```

Comment:
All other previous info logs will be JSON formatted, right?

Reply:
yes